app/views/users/show.html.erb

```diff
@@ -1,29 +1,35 @@
 <%= title('Dashboard') %>
 
 <div class="container tmp-container-spacer">
-  <div class= 'col-lg-8 offset-lg-2'>
-    <h1 class="page-heading-title">Dashboard</h1>
+  <div class="grid grid-cols-12">
+    <div class= "col-span-12 md:col-span-8 md:col-start-3">
+      <h1 class="page-heading-title">Dashboard</h1>
 
-    <div class="card-main profile-card">
-      <%= render User::AvatarComponent.new(current_user: current_user, classes: ' w-24') %>
-      <h2 class="profile-card__username text-2xl pl-4" data-test-id="profile_username"><%= current_user.username %></h2>
-      <p class="profile-card__learning-goal" data-test-id='learning_goal'><%= display_dashboard_learning_goal(current_user).html_safe %></p>
-    </div>
-
-    <div class="skills">
-      <div class="skills__header">
-        <h2 class="skills__title text-2xl">Skills Progress</h2>
-        <% if current_user.path.courses.size > 1 %>
-          <p class="skills__content">
-            The following courses should be taken in order
-          </p>
+      <%= render CardComponent.new(classes: "mb-24 odin-dark-bg-accent") do |card| %>
+        <% card.body do %>
+          <div class="flex items-center">
+            <%= render User::AvatarComponent.new(current_user: current_user, classes: " w-24") %>
+            <h2 class="basis-3/5 text-2xl pl-4" data-test-id="profile_username"><%= current_user.username %></h2>
+            <p data-test-id='learning_goal'><%= display_dashboard_learning_goal(current_user).html_safe %></p>
+          </div>
         <% end %>
+      <% end %>
+
+      <div id="skills" class="mb-24">
```

Comment:
Just for consistency, can we use our test id data attribute instead of an id please?

```suggestion
      <div id="skills" class="mb-24" data-test-id="skills">
```